README.md

```diff
@@ -50,3 +50,10 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+##Configuring Trello API
+First you need to create an account https://trello.com/signup
+Then you need to generate API key and token (explained here https://trello.com/app-key). The link will display your key and you need to generate manually a token.
+Add them to .env file as variables TOKEN = {Your token} and KEY = {Your key}.
```

Comment:
To make things easier, you could put the variables (without real values) into .env.template